src/main/java/cms/gov/madie/measure/models/Measure.java

```diff
@@ -62,6 +62,9 @@ public class Measure {
   private String measureName;
 
   private boolean active = true;
+  // TODO: determine if theres a way to set this from backend or if we should always trust user
+  // input for this field
+  private boolean cqlErrors;
```

Comment:
Either we need to validate CQL for both parsing and translation errors in the backEnd or trust.

I believe this will also be the case for test case execution results, we do the execution in UI and so once we get to a point where we have to save those results, then we have to trust UI or run the execution again.

Reply:
Yeah, agreed. I just wanted to capture it for later discussion. I was also going to add a single tech debt item for this as well as to review other places were we implicitly trust all user input and see if there's anywhere we should stop doing that.